src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -136,68 +129,241 @@
     private static final double LOG_ODDS_THRESHOLD = 3.0;
 
     /** Metrics for evaluating the performance of whole genome sequencing experiments. */
-    public static class WgsMetrics extends MetricBase {
+    public static class WgsMetrics extends MergeableMetricBase {
+
+        @NoMergingIsDerived
+        protected final Histogram<Integer> depthHistogram;
+
+        @NoMergingIsDerived
+        protected final Histogram<Integer> baseQHistogram;
+
+        @NoMergingIsDerived
+        protected final int coverageCap;
+
+        @NoMergingIsDerived
+        protected final int sampleSize;
```

Comment:
I could turn my sample size really low since I don't care about baseq until I run the final `calculateDerivedFields`.  Het sensitivity is slow for larger sample sizes, especially when not needed.  I'll add a `@NoMerging` annotation.